src/control/server/server_utils.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/daos-stack/daos/src/control/server/config"
 	"github.com/daos-stack/daos/src/control/server/engine"
 	"github.com/daos-stack/daos/src/control/server/storage"
+	"github.com/daos-stack/daos/src/control/server/storage/bdev"
```

Comment:
The server package should not depend directly on these sub-packages.

Reply:
It looks like FaultBdevNotFound is what pulled it in. Maybe that fault could be moved to the storage package?